src/main/java/step2/state/StrikeSymbol.java

```diff
@@ -0,0 +1,51 @@
+package step2.state;
+
+import step2.domain.dto.PointDTO;
+
+import java.util.Objects;
+
+import static step2.state.Symbol.*;
+
+public class StrikeSymbol implements Symbol {
```

Comment:
Strike, Spare 등의 객체들은 Symbol을 구현함을 시그니처로 알 수 있기 때문에 각 상태 네이밍에서 Symbol을 제거해도 괜찮을 것 같아요 :)
```suggestion
public class Strike implements Symbol {
```